### ISSUE TITLE
Utilize Open Graph Meta Tags

### ISSUE THREAD

**Issue by voidstar0** [alii/blog#3]
[Open Graph meta tags](https://ogp.me/) provide useful information to sites looking to embed posts from your site. Information such as the title, author, publication date, etc.. are already present in the `Post` class so the change would be quite simple. 

If this is a change you'd like, I can create the PR for this.

**Comment by alii** [comment]
Brilliant idea! If you would like to create a PR, that would be totally awesome! :)

**Comment by nexxeln** [comment]
I can create a PR for this. @alii do you also want images related to the post to be in the embeds?

PS: Really good blog, I will use it as mine too. thanks :)

**Comment by alii** [comment]
Yeah that would be cool. Although perhaps it would be better if we manually specified which image to use in the post's config. You can type it as `NextStaticImage` (or something like that) and access via `.src` which would allow the following

```ts
import image from './image.png';

export class MyBoringBlogPost extends Post {
	public ogImage = image;
	// ...
}
```

**Comment by nexxeln** [comment]
Yeah I was thinking along the same lines. I'll try to implement this tomorrow (12 am here rn).

**Comment by alii** [comment]
no rush! thank you so much :)
